ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable.yaml

```diff
@@ -68,6 +72,31 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
+- as: aws-ipi-byo-route53-upgrade-rollback-oldest-supported-f28
```

Comment:
I feel like it is better to have a separate ci yaml to run the rollback upgrade job for easy management.

Reply:
Actually I am also thinking if we can split `to-multiarch` jobs to a separate ci yaml for the same reason.

Reply:
Updated. Thanks.